### DISCUSSION TITLE
Roundcube - critical security update (1.6.4)

### DISCUSSION THREAD

**Discussion by jcz1** [Mailu/Mailu#3022] [User support] [answered]
Hello.

There is a security update of Roundcube:

https://github.com/roundcube/roundcubemail/releases/tag/1.6.4

https://www.bleepingcomputer.com/news/security/european-govt-email-servers-hacked-using-roundcube-zero-day/

Please what is update policy for components (roundcube + modules, postfix, dovecot, ... + libraries)? Because this security update is almost 2 weeks old.

Thanks.

**Comment by Diman0** [comment, marked as answer]
This was addressed two weeks ago
https://github.com/Mailu/Mailu/releases/tag/2.0.29

If you use docker-compose, then the images are not automatically updated. You must run `docker compose pull` to pull the new images and then run `docker compose up -d` to use the new images.

**Comment by jcz1** [comment]
Thanks for information. I'm sorry, I checked a release page earlier, but probably bad page or something like that.
